stack.yml

```diff
@@ -0,0 +1,25 @@
+# Use postgres/example user/password credentials
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: example
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
+
+  uat:
+    image: uat
+    restart: always
+    ports:
+      - 9000:9000
+    volumes:
+      - ./universal-application-tool-0.0.1:/code
+    command: sbt ~run
```

Comment:
I think you need to add links - but other than that, LGTM!
https://docs.docker.com/compose/networking/#links

Reply:
Done. I am still figuring out how to use GitHub for code review so definitely let me know if I am doing something wrong/weird.

Reply:
Nope, that's exactly right! :)